### DISCUSSION TITLE
integration with spaces with digital ocean

### DISCUSSION THREAD

**Discussion by abcnow** [serverpod/serverpod#713] [Ideas]
I noticed that you can integrate serverpod with s3 and google cloud .. I wonder if it is possible to integrate it with spaces from digitalocean --out of all options it is the most cost effective :) -> https://www.digitalocean.com/products/spaces

**Comment by vlidholt** [comment]
There is no pre-made Terraform script for Digital Ocean yet, but Serverpod will run anywhere you can run Dart and Postgres.
